src/payment/AlertCodeMessages.jsx

```diff
@@ -1,9 +1,14 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { FormattedMessage } from '@edx/frontend-i18n';
+import { FormattedMessage } from '@edx/frontend-platform/i18n';
 import { Hyperlink } from '@edx/paragon';
 import LocalizedPrice from './cart/LocalizedPrice';
-import { App } from '@edx/frontend-base';
+import { getConfig, mergeConfig } from '@edx/frontend-platform';
+
+
+mergeConfig({
```

Comment:
This is duplicative of the `mergeConfig`in index.jsx above.   I think this one can be deleted.

Reply:
It's added so tests work. I could put this code in the test itself. wdyt?

Reply:
Ooohhh, I see.  Huh.  I wonder if you could do it once in setupTest to mirror what happens in index.jsx?

Reply:
Oh wow yea. I'll do that.